### ISSUE TITLE
Show JASP's HTML results output in RStudio viewer

### ISSUE THREAD

**Issue by JorisGoosen** [jasp-stats/jasp-issues#216]
It would be nice to be able to see the results html, like in JASP, in RStudio.

This way it becomes much much easier to develop an analysis and should be simple enough with jaspResults. We could integrate the html/js/etc into the package.

**Comment by JorisGoosen** [comment]
@boutinb we should have a look at this when we are running JASP in R anyway ^^

Is there already a project for syntax mode? No right?
How are you collecting issues for that version because we ought to add this one to it.

**Comment by boutinb** [comment]
> Is there already a project for syntax mode? No right?

Yes we have the Syntax Mode project, but it was not linked with the jasp-issues repo. I have just done it.

**Comment by JorisGoosen** [comment]
Merci
